From 92cb135855b71333a77aabe7b5364189c66bf07b Mon Sep 17 00:00:00 2001
From: Seija Kijin <doremylover123@gmail.com>
Date: Wed, 14 Dec 2022 16:52:23 +0000
Subject: [PATCH 0570/2871] git: remove duplicate includes

These files are already included; we do not need to include them again

Signed-off-by: Seija Kijin <doremylover123@gmail.com>
Acked-by: Jeff King <peff@peff.net>
Signed-off-by: Junio C Hamano <gitster@pobox.com>
---
 pack-write.c   | 1 -
 sequencer.c    | 1 -
 unpack-trees.c | 1 -
 3 files changed, 3 deletions(-)

diff --git a/pack-write.c b/pack-write.c
index 00787e306d..3363729748 100644
--- a/pack-write.c
+++ b/pack-write.c
@@ -5,7 +5,6 @@
 #include "chunk-format.h"
 #include "pack-mtimes.h"
 #include "oidmap.h"
-#include "chunk-format.h"
 #include "pack-objects.h"
 
 void reset_pack_idx_option(struct pack_idx_option *opts)
diff --git a/sequencer.c b/sequencer.c
index 9827e1adcb..f51d403b7d 100644
--- a/sequencer.c
+++ b/sequencer.c
@@ -36,7 +36,6 @@
 #include "rebase-interactive.h"
 #include "reset.h"
 #include "branch.h"
-#include "log-tree.h"
 
 #define GIT_REFLOG_ACTION "GIT_REFLOG_ACTION"
 
diff --git a/unpack-trees.c b/unpack-trees.c
index 8a762aa077..d11bef96bf 100644
--- a/unpack-trees.c
+++ b/unpack-trees.c
@@ -19,7 +19,6 @@
 #include "promisor-remote.h"
 #include "entry.h"
 #include "parallel-checkout.h"
-#include "sparse-index.h"
 
 /*
  * Error messages expected by scripts out of plumbing commands such as
-- 
2.39.3 (Apple Git-145)

